test/e2e/atlas/clusters_file_test.go

```diff
@@ -128,6 +128,21 @@ func TestClustersFile(t *testing.T) {
 		require.NoError(t, err, string(resp))
 	})
 
+	t.Run("Create index with unknown fields", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			clustersEntity,
+			indexEntity,
+			"create",
+			"--clusterName", clusterFileName,
+			"--file=data/create_index_test-unknown-fields.json",
+			"--projectId", g.projectID,
+		)
+
+		cmd.Env = os.Environ()
+		resp, _ := cmd.CombinedOutput()
```

Comment:
it would be better to make sure the error only shows on stderr and doesn't leak to stdout